engine/class_modules/sc_death_knight.cpp

```diff
@@ -7782,15 +7783,18 @@ void death_knight_t::default_apl_frost()
   standard -> add_action( this, "Remorseless Winter", "", "Standard single-target rotation" );
   standard -> add_action( this, "Frost Strike", "if=cooldown.remorseless_winter.remains<=2*gcd&talent.gathering_storm.enabled" );
   standard -> add_action( this, "Howling Blast", "if=buff.rime.up" );
+  standard -> add_action( this, "Frostscythe", "if=talent.icecap.enabled&buff.killing_machine.up" );
```

Comment:
Please use buff.killing_machine.react to account for the reaction time after a proc.

Reply:
updated to reflect this change.